README.md

```diff
@@ -7,7 +7,7 @@ You're working on your Tunr app and you encounter this error. What does it mean
 ![Rails error](http://i.imgur.com/9NR7XNT.png)  
 
 ```text
-Your answer...
+Error happens if you don't redirect the create method in the controller...
```

Comment:
i think this one is missing the template - we know because it looks for index template